modules/wri_person/config/install/core.entity_form_display.node.person.default.yml

```diff
@@ -236,6 +238,26 @@ content:
       sidebar: true
       use_details: true
     third_party_settings: {  }
+  field_paragraphs:
+    type: paragraphs
+    weight: 4
+    region: content
+    settings:
+      title: 'Social Link'
+      title_plural: 'Social Links'
+      edit_mode: closed
+      closed_mode: summary
+      autocollapse: none
+      closed_mode_threshold: 0
+      add_mode: button
+      form_display_mode: default
+      default_paragraph_type: social_media_sites
```

Comment:
Can you change these to `default_paragraph_type: _none`? Otherwise it's not clear that the social site isn't required (since fields within it are).